test/index.js

```diff
@@ -340,6 +341,25 @@ describe('Feed generator', () => {
       feed_url: hexo.config.root + feedCfg.path
     }));
   });
+
+  it('no updated date', () => {
+    hexo.config.optional_updated = true;
```

Comment:
Most probably this config won't be applied as [`processor`](https://github.com/hexojs/hexo/blob/master/lib/plugins/processor/asset.js) doesn't run here.